tests/test-vic-serial.c

```diff
@@ -32,7 +67,7 @@ static char * all_tests()
 int main(void)
 {
 
-        vic_init();
+    vic_init();
```

Comment:
@vargac it seems like this uses a different amount of spaces for a tab (4 versus 8).